blueprint/src/chapter/pfr-entropy.tex

```diff
@@ -230,7 +230,7 @@ \subsection{Endgame}
 
 \begin{lemma}[Bound on conditional mutual informations]\label{uvw-s}
 \uses{conditional-information-def}
-\lean{sum_condMutual_le}\leanok
+\lean{I₃_eq,sum_condMutual_le}\leanok
```

Comment:
I've never used blueprints before, so I'm guessing that this is how to add this lemma as an extra dependency.